src/Pictures/favicon.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
The Pictures folder is only for images, this html file doesn't help us in the main codebase.